src/main/java/mocacong/server/controller/CafeController.java

```diff
@@ -77,4 +79,16 @@ public ResponseEntity<CafeFilterResponse> getCafesByStudyType(@RequestParam(requ
         CafeFilterResponse responseBody = cafeService.filterCafesByStudyType(studytype, requestBody);
         return ResponseEntity.ok(responseBody);
     }
+
+    @Operation(summary = "카페 이미지 업로드")
+    @SecurityRequirement(name = "JWT")
+    @PostMapping(value = "/{mapId}/img", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<String> updateCafeImage(
```

Comment:
반환해주는 response가 없는 경우라면 `Void` 타입 반환은 어떤가요?
```suggestion
    public ResponseEntity<Void> updateCafeImage(
```